### DISCUSSION TITLE
Setting up Podman on server in local network to connect with

### DISCUSSION THREAD

**Discussion by HappiePlant** [marhkb/pods#731] [Q&A] [answered]
Hello!
I've been trying to use this wonderful app to connect to podman on my mini server in my local network, I don't need to root mode or anything but I have no clue what port number I need or if I need to setup anything. I don't have much experience with Podman.

**Comment by marhkb** [comment, marked as answer]
> this wonderful app

Thank you!

> no clue what port number

You can use any port that's available and to that you connect through your local network.

On your mini server, you can for example run 
```
podman system service --time=0 tcp://0.0.0.0:37017
```

With pods you can then connect

![image](https://github.com/marhkb/pods/assets/3630213/18990180-4507-44fe-b6f3-96112c5f0fc9)

You should also probably create and enable a systemd unit to start the podman service automatically
```
[Unit]
Description=Podman TCP socket
After=network.target
StartLimitIntervalSec=0

[Service]
Type=simple
Restart=always
RestartSec=1
ExecStart=/usr/bin/podman system service --time=0 tcp://0.0.0.0:37017

[Install]
WantedBy=multi-user.target
```

**Reply by HappiePlant** [reply]
Thank you for your reply! It turns out firewalld wasn't allowing connections to the port number (oops :sweat_smile:) and podman also wasn't working. But it works now! (Your reply was at the bottom of my GitHub notifications and I just saw it :))
